mirdata/initialize.py

```diff
@@ -5,6 +5,15 @@
 from mirdata import DATASETS
 
 
+def list_datasets():
+    """Get a list of all mirdata dataset names
+
+    Returns:
+        list: list of dataset names as strings
+    """
+    return DATASETS
+
+
```

Comment:
I find a bit confusing to do `mirdata.initialize.list_datasets` and also `mirdata.initialize(dataset)`. Also for some reason in the docs we're getting a lot of `mirdata.list_datasets.__dir__()`, `mirdata.list_datasets.__new__()` which we probably don't won't there?

I'd suggest we either simply use `mirdata.DATASETS`, or we move `list_datasets()` to `__init__.py` (i understand it's ok to have code there as in any other module?). I'm a bit more inclined for the second. Also, should we use `list_datasets` or `list_loaders`?

In any case, what about adding in the docs a small code-block at the top of `overview` and `tutorial` to show people they can easily see what loaders are available?